ceno_emul/src/host_utils.rs

```diff
@@ -0,0 +1,32 @@
+use std::iter::from_fn;
+
+use crate::{ByteAddr, EmuContext, VMState, WordAddr};
+
+const WORD_SIZE: usize = 4;
+const INFO_OUT_ADDR: WordAddr = ByteAddr(0xC000_0000).waddr();
```

Comment:
is this a placeholder for now until we have the public IO?

Reply:
It's a placeholder of sorts, but for debug output, not for public IO.

We have an open issue to improve debug output.